src/components/plot/chart-with-effect.tsx

```diff
@@ -146,33 +150,35 @@ const ChartWithEffect = ({ config }: ChartWithEffectProps): JSX.Element => {
   }
 
   return (
-    <div className='chart-container'>
-      <div className='chart'>
-        {selectData && selectData.length > 0 ? (
-          <SelectGrid selectors={selectData} width={dynamicStyles.width} />
-        ) : null}
-        {isParseLoading || isFetchSelected ? (
-          <ChaiseSpinner />
-        ) : (
-          <PlotlyChart
-            className='plotly-chart'
-            style={dynamicStyles}
-            ref={plotlyRef}
-            {...parsedData}
-            useResizeHandler
+    <AlertsProvider>
```

Comment:
`AlertsProvider` doesn't need to be added here. Since we are using `AppWrapper` (and setting `includeAlerts`), the `AlertsProvider` will be automatically included